caffe2/operators/experimental/c10/schemas/add.cc

```diff
@@ -23,6 +23,7 @@ C10_DEFINE_OP_SCHEMA(
 
 namespace caffe2 {
 REGISTER_C10_OPERATOR_FOR_CAFFE2_DISPATCH_CPU(
-    ops::Add(),
+    "_c10_experimental::Add",
+    "",
```

Comment:
(and I am not the only person to complain about this)

Reply:
Yeah, why can't we parse the ".overload" syntax instead?

Reply:
Yeah, that would be so much better!

Reply:
Yes, I was planning to parse it from a "op.overload" syntax string, but punted on the implementation because it is kind of connected to the function schema parser and I don't have access to that from c10 yet.

Will for now just remove the empty string and disallow exposing non-empty overloads from c10 to caffe2.